contributing-guides/style-guide.md

```diff
@@ -66,14 +66,18 @@ Keep the following guidelines in mind when choosing tokens:
 - In case of a possible reference both to a file or a directory,
   use `{{path/to/file_or_directory}}`.
 
-Extensions
+### Extensions
 
 - If a particular extension is expected for the file, append it.
   For example, `unrar x {{compressed.rar}}`.
 - In case a generic extension is needed, use `{{.ext}}`, but **only** if an extension is required.
   For instance, in `find.md`'s example "Find files by extension" (`find {{root_path}} -name '{{*.ext}}'`)
   using `{{*.ext}}` explains the command without being unnecessarily specific;
   while in `wc -l {{file}}` using `{{file}}` (without extension) is sufficient.
+  
+### Options
+
+- Prefer long options instead of short ones whenever is possible.
```

Comment:
I think they're mostly referred to as "flags".
```suggestion
### Flags

- Prefer long flags instead of short ones whenever it is possible. E.g. use `--command`, instead of `-c`. This makes it easier for the user to remember and understand them.
```

Reply:
As I know flags are options without value. That's why I used `options` term.

Reply:
Options is definitely not a good fit, if I was new, I would interpret that as being the token syntax at a glance

Reply:
I'd reword this to:

```suggestion
### Flags

- Always use long flags when applicable, for example instead of `bash -c`, use `bash --command`, so the reader can understand the meaning of a flag.
```